assayist/common/models/content.py

```diff
@@ -106,3 +108,20 @@ class Checksum(AssayistStructuredNode):
     artifacts = RelationshipTo('Artifact', 'CHECKSUMS')
     # The external artifacts this checksum is associated with
     external_artifacts = RelationshipTo('ExternalArtifact', 'CHECKSUMS')
+
+
+class UnknownFile(AssayistStructuredNode):
+    """
+    The definition of an UnknownFile node.
+
+    Any unclaimed files by analyzers will get UnknownFile nodes created with edges to the artifact
+    it's a part of.
+    """
+    # SHA256 checksum of the file
```

Comment:
Is this what TravisCI is barking about?

Reply:
yep, flake8 complains here.

```
./assayist/common/models/content.py:113:1: D204 1 blank line required after class docstring
```

@mprahl please fix that and then you have another +1 from me.

Reply:
Same from me.